### PR TITLE
Fix #396

### LINKED ISSUE #396: form-control in some behavior fields missing
https://github.com/plone/plone.app.event/blob/a98840d2e574d921df7f01718a41dc550641951e/plone/app/event/dx/behaviors.py#L30C1-L32C60

the widgets should be imported from plone.app.z3cform, not from z3c.form package.

Fields:
- location
- attendees
- contact_name
- contact_email
- contact_phone
- event_url

### PR DESCRIPTION
**Pull request by 1letter**
change imports of some widgets to `plone.app.z3cform`, make widget bootstrap 5 ready

### CODE REVIEW

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

File: plone/app/event/dx/behaviors.py [hunk 1]
```diff
@@ -41,6 +39,23 @@
...
+except ImportError:
+    from z3c.form.browser.text import TextFieldWidget
+
+try:
+    # Import fails for Plone < 6.1
+    # version pin of plone.app.z3cform is set to plone.app.z3cform==4.3.2
+    # TextLinesFieldWidget is introduced in plone.app.z3cform==4.4.0
+    from z3c.form.browser.textlines import TextLinesFieldWidget
+except ImportError:
+    from z3c.form.browser.textlines import TextLinesFieldWidget
```
Comment (petschki) [inline plone/app/event/dx/behaviors.py:56]: This is the same import after `ImportError`

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

File: plone/app/event/dx/behaviors.py (hunk 1 above)
Comment (1letter) [reply plone/app/event/dx/behaviors.py:56]: ah sorry... i updated the import snippet and started the tests

**Review by petschki — APPROVED** [review]
merge when green 👍🏼

**Pull request merged**